### PR TITLE
Make galaxy lint happy

### PR DESCRIPTION
**Pull request by tiraboschi**

### CODE REVIEW

File: tasks/restore_backup.yml [hunk 1]
```diff
@@ -18,15 +21,20 @@
     path: /root/engine_backup
 - include_tasks: pg_scl.yml
 - name: Remove previous hosted-engine VM
-  command: "{{ scl_pg_prefix }} psql -d engine -c \"SELECT deletevm(vm_guid) FROM (SELECT vm_guid FROM vms WHERE origin=6) t\""
+  command:
```
[the remark this answers is not in the document]
Comment (tiraboschi) [reply tasks/restore_backup.yml]: Done

File: tasks/initial_clean.yml [hunk 2]
```diff
@@ -22,16 +25,29 @@
...
+          src: /usr/share/abrt/conf.d/plugins/vmcore.conf,
+          dest: /etc/abrt/plugins/vmcore.conf
+        }
   - name: Restart abrtd service
     service:
       name: abrtd
       state: restarted
   - name: Drop libvirt sasl2 configuration by vdsm
-    command: sed -i '/## start vdsm-4.[0-9]\+.[0-9]\+ configuration/,/## end vdsm configuration/d' /etc/sasl2/libvirt.conf
+    command:
```
Comment (irosenzw) [inline tasks/initial_clean.yml]: here too. sorry I missed it.

File: tasks/create_target_vm/03_hosted_engine_final_tasks.yml [hunk 3]
```diff
@@ -102,7 +128,8 @@
     set_fact: spmId="{{ hostvars[he_fqdn]['host_spm_id'] }}"
   - debug: var=spmId
   - name: Detect ovirt-hosted-engine-ha version
-    command: python -c 'from ovirt_hosted_engine_ha.agent import constants as agentconst; print(agentconst.PACKAGE_VERSION)'
+    command:
```
Comment (irosenzw) [inline tasks/create_target_vm/03_hosted_engine_final_tasks.yml]: here too.

File: tasks/initial_clean.yml (hunk 2 above)
Comment (tiraboschi) [reply tasks/initial_clean.yml]: In theory '>-' is neede only if we break over multiple lines but let's try to keep it more consistent

**Pull request merged**